nncf/quantization/algorithms/smooth_quant/torch_backend.py

```diff
@@ -89,7 +89,7 @@ def get_activations_port_id(node: NNCFNode, nncf_graph: NNCFGraph) -> int:
 
     @staticmethod
     def get_channel_agnostic_reduction_axes(channel_axis: int, shape: Tuple[int]) -> Tuple[int]:
-        return get_channel_agnostic_reduction_axes([channel_axis], shape)
```

Comment:
The `get_channel_agnostic_reduction_axes` was removed from the `SmoothQuantAlgoBackend` and OpenVINO backend as well. But was not removed from the Torch backend. Why?